Makefile

```diff
@@ -97,6 +101,17 @@ $(EBPF_PROBES) $(EBPF_HOOKS): $(LIBBPF_INCLUDES)
 	CFLAGS_INCLUDES="$(INCLUDES)" \
 	$(MAKE) -r -f $(ROOT_DIR)/ebpf.mk -C $@
 
+pylib:
+	$(call out_console,MATURIN,Building python bindings ...)
+	$(CONTAINER_RUNTIME) run --rm --name maturin -v $$PWD:/io:z ghcr.io/pyo3/maturin build -m retis-events/Cargo.toml -F python-lib  
```

Comment:
Isn't that redundant? The user knows they are building retis, right?

Reply:
Right; I was thinking about cases where the container stays around for whatever reason. If that happens it gives a clear hint to the user about where this is coming from, as well as not preventing future builds of Retis because of a name conflict.

That might be overthinking so feel free to ignore, that is not a strong comment.

Reply:
Oh, I thought you were referring to the name of the make target, the name of the container could use the word "retis", you're right.